### PR TITLE
Include eyb leads for investment projects in open search

### PR DESCRIPTION
**Pull request by marijnkampf**
### Description of change

Include EYB leads for investment projects in open search

### Checklist

* [x] Has this branch been rebased on top of the current `main` branch?

  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>

* [ ] Is the CircleCI build passing?

### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/main/docs/CONTRIBUTING.md) for more guidelines.

### CODE REVIEW

File: datahub/search/company_activity/test/test_dict_utils.py [hunk 1]
```diff
@@ -59,6 +60,42 @@ def test_activity_investment_dict():
...
+
+    investment = InvestmentProjectFactory()
+    eyb_leads_is_high_value = EYBLeadFactory(
+        investment_projects=[investment.id],
+        is_high_value=True,
+    )
+    eyb_leads_is_not_high_value = EYBLeadFactory(
+        investment_projects=[investment.id],
+        is_high_value=False,
+    )
```
Comment (swenban) [inline datahub/search/company_activity/test/test_dict_utils.py:89]: We should probably also ensure we are able to handle rare cases in which is_high_value is set to null

**Review by swenban — APPROVED** [review]
I've left a comment around testing but I have manually tested this with the frontend and it is behaving as expected so won't block you from merging

File: datahub/search/company_activity/dict_utils.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ def activity_investment_dict(obj):
         'number_new_jobs': obj.number_new_jobs,
         'created_by': dict_utils.contact_or_adviser_dict(obj.created_by),
         'client_contacts': dict_utils.contact_job_list_of_dicts(obj.client_contacts),
+        'eyb_leads': dict_utils.eyb_lead_list(obj.eyb_leads) if obj.eyb_leads else None,
```
Comment (DeanElliott96) [inline datahub/search/company_activity/dict_utils.py:55]: Would the eyb_leads for an investment need a prefetch_related? https://github.com/uktrade/data-hub-api/blob/91c9bf7be6d3fb32d85695a343a25e6f144c6b67/datahub/search/company_activity/apps.py#L15

Comment (marijnkampf) [reply datahub/search/company_activity/dict_utils.py:55]: That could be worthwhile adding. I've added `eyb_leads` to `prefetch_related` in datahub/search/investment/apps.py.

Comment (marijnkampf) [reply datahub/search/company_activity/dict_utils.py:55]: Fixed the fix in datahub/search/company_activity/apps.py

**Pull request merged**